pkg/log/zap/flags.go

```diff
@@ -26,6 +26,13 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	logLevelDebug string = "debug"
+	logLevelInfo  string = "info"
+	logLevelError string = "error"
+	logLevelPanic string = "panic"
+)
```

Comment:
Should not be better we use here the "github.com/sirupsen/logrus" levels definition instead of creating our own const as it was done [here](https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/bundle/internal/result.go#L93)